eng/scripts/Validate-TypeSpec.ps1

```diff
@@ -5,7 +5,9 @@ param (
   [Parameter(Position = 1, Mandatory = $false)]
   [string]$TargetBranch,
   [Parameter(Position = 2, Mandatory = $false)]
-  [string]$SourceBranch
+  [string]$SourceBranch,
+  [Parameter(Position = 3, Mandatory = $false)]
```

Comment:
```suggestion
  [Parameter(Mandatory = $false)]
```

If powershell allows it, I think we'd want this parameter to only work by name, and not position, right?

Reply:
fair point, let me do some testing first before I push